apps/storybook-react/.storybook/preview.tsx

```diff
@@ -21,18 +22,27 @@ export const globalTypes = {
   },
 };
 
-function withColorScheme(Story, context) {
-  const { colorScheme } = context.globals;
+function withColorScheme(Story: StoryFn, context: StoryContext) {
+  const storyColorScheme = context.parameters.colorScheme;
+  const globalColorScheme = context.globals.colorScheme;
 
-  function Wrapper(props) {
+  // Use story-level parameter if available, otherwise fall back to global
+  const colorScheme = storyColorScheme || globalColorScheme;
+
+  function Wrapper({
+    children,
+    ...props
+  }: { children: React.ReactNode } & Record<string, any>) {
```

Comment:
Fixing types issues in preview file

<img width="503" alt="Screenshot 2024-11-21 at 5 58 09 PM" src="https://github.com/user-attachments/assets/ef8e52c3-1976-4308-b0b6-0977eba83143">